browser/src/control/Control.UIManager.js

```diff
@@ -1710,7 +1710,13 @@ L.Control.UIManager = L.Control.extend({
 	enableTooltip: function(element) {
 		var elem = $(element);
 		if (window.mode.isDesktop()) {
-			elem.tooltip();
+			if (this._tooltip) {
+				this._tooltip.tooltip();
+				this._tooltip.tooltip('close');
+				this._tooltip = undefined;
+			}
+
+			this._tooltip = elem.tooltip();
```

Comment:
can we do the same in below "else" block? (looks similar case).
I'm sorry for that many comments :)

Reply:
Szymon I've modified the else block but ı've detected some difference between mine and your version. Let me send screen record

Reply:
When we remove all the tooltips with $(".ui-tooltip").remove(); 
Tooltip is less blinking 
[gulsah.webm](https://github.com/CollaboraOnline/online/assets/2954981/2b0a50dd-319b-478c-967d-0a7e520ffbab)


But if we use this._tooltip.tooltip('close'); We see more tooltip blinking:
[szymon.webm](https://github.com/CollaboraOnline/online/assets/2954981/901a93be-d9f8-4bae-847a-1a9fe9157cf4)


Should i use first version in the if block. What do you think?

Reply:
yes, please go with the best version you had.